packages/assets-controllers/src/MultichainBalancesController/constants.ts

```diff
@@ -0,0 +1,44 @@
+import { BtcAccountType, SolAccountType } from '@metamask/keyring-api';
+
+/**
+ * The network identifiers for supported networks in CAIP-2 format.
+ * Note: This is a temporary workaround until we have a more robust
+ * solution for network identifiers.
+ */
+export enum MultichainNetworks {
+  Bitcoin = 'bip122:000000000019d6689c085ae165831e93',
+  BitcoinTestnet = 'bip122:000000000933ea01ad0ee984209779ba',
+  Solana = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
+  SolanaDevnet = 'solana:EtWTRABZaYq6iMfeYKouRu166VU2xqa1',
+  SolanaTestnet = 'solana:4uhcVJyU9pJkvQyS88uRDiswHXSCkY3z',
+}
+
+export enum MultichainNativeAssets {
+  Bitcoin = `${MultichainNetworks.Bitcoin}/slip44:0`,
+  BitcoinTestnet = `${MultichainNetworks.BitcoinTestnet}/slip44:0`,
+  Solana = `${MultichainNetworks.Solana}/slip44:501`,
+  SolanaDevnet = `${MultichainNetworks.SolanaDevnet}/slip44:501`,
+  SolanaTestnet = `${MultichainNetworks.SolanaTestnet}/slip44:501`,
+}
+
+const BTC_AVG_BLOCK_TIME = 10 * 60 * 1000; // 10 minutes in milliseconds
```

Comment:
Since we are naming Solana like `SOLANA`, I'd rename this one to:
```suggestion
const BITCOIN_AVG_BLOCK_TIME = 10 * 60 * 1000; // 10 minutes in milliseconds
```

Reply:
Updated in [017c657](https://github.com/MetaMask/core/pull/4965/commits/017c657b1fe8cee52a607d13a4186947f4135f54)